src/main.cpp

```diff
@@ -40,6 +41,8 @@ static const string hsm_tls_key_id = "01";           // TLS key ID on HSM, when
 static const string hsm_client_cert_id = "03";       // Client certificate's ID on HSM, when used (for sota.toml)
 static const string hsm_tls_key_label = "tls";       // TLS key label on HSM, when used (for OpenSSL)
 static const string hsm_client_cert_label = "client"; // Uptane key label on HSM, when used (for OpenSSL)
+static const string installed_versions_filename = "/var/sota/import/installed_versions";
```

Comment:
I'm still not seeing under what conditions our ability to read this file is going to be helpful? Especially when the next commit is going to require os-release or the user will have to manually provide this type info for the tag?

Reply:
@mwasilew I don't have a strong opinion on parsing `installed_versions`, so feel free to remove it and do as Andy suggests.

My only argument was that it's more reliable to pass data via json and parse it, compared to the key-value text file. This is very weak argument, so the simplicity wins :).